unittests/InstructionCountCI/AFP/VEX_map1.json

```diff
@@ -10,27 +10,29 @@
   },
   "Instructions": {
     "vsqrtss xmm0, xmm1, xmm2": {
-      "ExpectedInstructionCount": 2,
+      "ExpectedInstructionCount": 3,
       "Comment": [
         "Map 1 0b10 0x51 128-bit"
       ],
       "ExpectedArm64ASM": [
         "mov v16.16b, v17.16b",
-        "fsqrt s16, s18"
+        "fsqrt s0, s18",
```

Comment:
Also confused by this change. Pseudocode of fsqrt
```
boolean merge = IsMerging(FPCR);
bits(128) result = if merge then V[d, 128] else 0<127:0>;
```
We already move xmm1 in to xmm0 at the start, so then `fsqrt s16 (xmm0), s18 (xmm2)` should insert correctly here?

Same situation for the 64-bit version

Reply:
humm, I might have been too eager to get the proper upper bits. Also I misunderstood the spec to what happens without AFP.

Reply:
I am preparing a simpler patch for this.